lib/formatters/json.js

```diff
@@ -8,30 +6,14 @@
  * JSON formatter.
  * @public
  * @function formatJSON
- * @param    {Object} req  the request object
+ * @param    {Object} req  the request object (not used)
  * @param    {Object} res  the response object
  * @param    {Object} body response body
  * @param    {Function} cb cb
  * @returns  {String}
  */
 function formatJSON(req, res, body, cb) {
-    if (body instanceof Error) {
-        // snoop for RestError or HttpError, but don't rely on
-        // instanceof
-        res.statusCode = body.statusCode || 500;
-
-        if (body.body) {
-            body = body.body;
-        } else {
-            body = {
-                message: body.message
-            };
-        }
-    } else if (Buffer.isBuffer(body)) {
-        body = body.toString('base64');
-    }
-
-    var data = JSON.stringify(body);
+    var data = body ? JSON.stringify(body) : '';
```

Comment:
The assumption here is that `data` would be a valid JSON string, which would mean it would need to be: `'""'` in order to be a valid.